core/tests/protractor/cacheSlugs.js

```diff
@@ -0,0 +1,49 @@
+// Copyright 2016 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview End-to-end tests for cache slugs.
+ */
+
+var general = require('../protractor_utils/general.js');
+
+var ERROR_PAGE_URL_SUFFIX = '/console-errors';
```

Comment:
The page is named /console_errors. This checks /console-errors (hyphen instead of underscore). I'm worried that it's not testing what you expect to test.

More worryingly, why wasn't this detected? Unless I'm missing something, I suspect there's an error in your tests, so please could you dig into this and add an explanation here of why the tests still passed despite looking at the wrong page? If the tests aren't testing the right thing, they aren't worth much...